From: Debian CLI Applications Team
 <pkg-cli-apps-team@lists.alioth.debian.org>
Date: Tue, 21 Jun 2011 19:43:31 +0200
Subject: fix XSL search path

search for XSL files in /usr/share/keepass2

Forwarded: not-needed
---
 KeePass/App/AppDefs.cs     | 2 ++
 KeePass/Forms/AboutForm.cs | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: keepass2/KeePass/App/AppDefs.cs
===================================================================
--- keepass2.orig/KeePass/App/AppDefs.cs
+++ keepass2/KeePass/App/AppDefs.cs
@@ -42,6 +42,8 @@ namespace KeePass.App
 		public static readonly Color ColorQualityHigh = Color.FromArgb(0, 255, 0);
 		public static readonly Color ColorQualityMid = Color.FromArgb(255, 255, 0);
 
+		public const string DebianDataDir = "/usr/share/keepass2/";
+
 		public static readonly string LanguagesDir = "Languages";
 
 		public static readonly string PluginsDir = "Plugins";
Index: keepass2/KeePass/Forms/AboutForm.cs
===================================================================
--- keepass2.orig/KeePass/Forms/AboutForm.cs
+++ keepass2/KeePass/Forms/AboutForm.cs
@@ -133,7 +133,7 @@ namespace KeePass.Forms
 			m_lvComponents.Columns.Add(strValueColumn, 100);
 
 			string strExe = WinUtil.GetExecutable();
-			string strDir = UrlUtil.GetFileDirectory(strExe, true, false);
+			string strDir = AppDefs.DebianDataDir;
 
 			AddComponentItem(PwDefs.ShortProductName, strMainVersion, strExe);
 
